client/src/components/VideoDisplay.js

```diff
@@ -0,0 +1,101 @@
+import { useState } from "react";
+import AddVideo from "./AddVideo";
+import SearchVideo from "./SearchVideo"
+
+const VideoDisplay = (prop) => {
+  const [allvideos, setAllVideos] = useState(prop.video);
+  const [searched, setsearchedVideos] = useState(prop.video);
+  const inputVideo = (newvideo) => {
+    setAllVideos([...allvideos].concat(newvideo));
+    setsearchedVideos([...allvideos].concat(newvideo));
+  };
+  const onsearch = (newvideo) => {
+    setAllVideos(newvideo);
+  };
+  
+  const [voteCount, setVoteCount] = useState(0);
+
+  const upVotes = (id) => {
+    [...allvideos].filter((video, index) => {
+      if (id === video.id) {
+        setVoteCount(video.rating+=1);
+      }
+    })
+   
+  };
+  const downVotes = (id) => {
+    [...allvideos].filter((video, index) => {
+      if (id === video.id)
+      {
+        setVoteCount(video.rating -= 1);
+      }
+    })
+   
+  };
+  const deleteVideo = (id) => {
+    setAllVideos([...allvideos].filter((video, index) => id !== video.id));
+  };
+
+  return (
+    <div className="render">
+      <div>
+        <AddVideo onClick={inputVideo} />
+        <SearchVideo videos={searched} onClick={onsearch} />
+      </div>{" "}
+      <div className="videos">
+        {[...allvideos]
+          .sort((a, b) => b.rating - a.rating)
+          .map((videos, index) => {
```

Comment:
Nice job sorting by rating! Instead of mapping a div below, you should think about breaking this into a separate component so that you return a component for each Video (each video component can then have its own voteCount state)